File: migration/pset6/cash.py
from cs50 import get_float

while True:
    number = get_float("Change owed: ")
    if number > 0:
        break
    
cont = 0
number = int(number * 100)
while True:
    if number == 0:
        break
    
    elif number >= 25:
        number -= 25
        cont += 1
        
    elif number >= 10:
        number -= 10
        cont += 1
        
    elif number >= 5:
        number -= 5
        cont += 1
    
    elif number >= 1:
        number -= 1
        cont += 1

    
print(cont)

File: migration/pset1/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float n; //
    int dollars, sum = 0;
    do
    {
        n = get_float("Change owed: "); //get float number
    }
    while (n <= 0); //get only positive numbers

    dollars = round(n * 100); //convert to integer
    
    while (dollars >= 25) //while the dollars is greater than 25
    {
        sum++; //"sum" receive one more
        dollars = dollars - 25; //dollars decreases 25
    }

    while (dollars >= 10)
    {
        sum++;
        dollars = dollars - 10;
    }
    
    while (dollars >= 5)
    {
        sum++;
        dollars = dollars - 5;
    }
    
    while (dollars >= 1)
    {
        sum++;
        dollars = dollars - 1;
    }
    
    printf("%i\n", sum); //print the total of coins
}